src/SmartCommander/Models/OptionsModel.cs

```diff
@@ -48,11 +48,12 @@ static OptionsModel()
 
         public bool IsMaximized { get; set; }
 
-        public string LeftPanePath { get; set; } = "";
+        public string LeftPanePath { get; set; } = string.Empty;
 
```

Comment:
I don't think that string.Empty is more readable. Would prefer to leave "" in all cases.

Reply:
fixed